aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -353,985 +355,37 @@ The name of the connection string is passed in to the context by calling a metho
 
 # [Visual Studio](#tab/visual-studio)
 
-In this section, the **Package Manager Console** (PMC) window is used to:
-
-* Add an initial migration.
-* Update the database with the initial migration.
-
-1. From the **Tools** menu, select **NuGet Package Manager** > **Package Manager Console**.
-
-   ![PMC menu](model/_static/5/pmc.png)
-
-1. In the PMC, enter the following commands:
-
-   ```powershell
-   Add-Migration InitialCreate
-   Update-Database
-
-   ```
+[!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie80/Program.cs?name=snippet_all&highlight=9-10)]
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-* In the **Solution Tool Window**, right-click the *RazorPagesMovie* project, and then select **Open in Terminal**. 
-
-  The **Terminal** window opens with the command prompt at the project directory, which contains the Program.cs and .csproj files.
-
-* Run the following .NET CLI commands:
-
-  ```dotnetcli
-  dotnet tool install --global dotnet-ef
-  dotnet ef migrations add InitialCreate
-  dotnet ef database update
-  ```
-
-> [!NOTE]
-> For SQLite, column type for the `Price` field is set to `TEXT`. This is resolved in a later step.
+[!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie80/Program.cs?name=snippet_all_sl&highlight=7-8)]
 
 ---
 
-The preceding commands install [the Entity Framework Core tools](/ef/core/get-started/overview/install#get-the-entity-framework-core-tools) and run the `migrations` command to generate code that creates the initial database schema.
-
-The following warning is displayed, which is addressed in a later step:
-
-> No type was specified for the decimal column 'Price' on entity type 'Movie'. This will cause values to be silently truncated if they do not fit in the default precision and scale. Explicitly specify the SQL server column type that can accommodate all the values using 'HasColumnType()'.
-
-The `migrations` command generates code to create the initial database schema. The schema is based on the model specified in `DbContext`. The `InitialCreate` argument is used to name the migrations. Any name can be used, but by convention a name is selected that describes the migration.
+## Troubleshooting with the completed sample
 
-The `update` command runs the `Up` method in migrations that have not been applied. In this case, `update` runs the `Up` method in the `Migrations/<time-stamp>_InitialCreate.cs` file, which creates the database.
+If you run into a problem you can't resolve, compare your code to the completed project. [View or download completed project](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie70) ([how to download](xref:index#how-to-download-a-sample)).
 
-### Examine the context registered with dependency injection
+## Additional resources
```

Comment:
Preexisting issue: On this and the subsequent pages, "Additional resources" without any actual additional resource links doesn't make sense when it only contains the "next" and "previous" links. There should be a "Next steps" h2 as in the first tutorial. If there are no actual "additional resources" links, that h2 can be omitted.

```suggestion
## Next steps
```